test/integration/scheduler_perf/config/scheduler-config-file.yaml.template

```diff
@@ -0,0 +1,77 @@
+apiVersion: kubescheduler.config.k8s.io/v1beta1
+clientConnection:
+  acceptContentTypes: ""
+  burst: 100
+  contentType: application/vnd.kubernetes.protobuf
+  kubeconfig: /etc/kubernetes/scheduler.conf
+  qps: 50
+enableContentionProfiling: true
+enableProfiling: true
+healthzBindAddress: ""
+kind: KubeSchedulerConfiguration
+leaderElection:
+  leaderElect: true
+  leaseDuration: 15s
+  renewDeadline: 10s
+  resourceLock: leases
+  resourceName: kube-scheduler
+  resourceNamespace: kube-system
+  retryPeriod: 2s
+metricsBindAddress: ""
```

Comment:
commented elsewhere: these attributes seem not to be respected properly. If so, let's remove them and add a comment to avoid confusion.

Reply:
yes, this is intentional, as I cannot see this is matter with the perf result, will sort it out.

Reply:
Done